cluster/gce/util.sh

```diff
@@ -1565,6 +1565,7 @@ KUBECONFIG_FILE: $(yaml-quote "${WINDOWS_KUBECONFIG_FILE}")
 BOOTSTRAP_KUBECONFIG_FILE: $(yaml-quote "${WINDOWS_BOOTSTRAP_KUBECONFIG_FILE}")
 KUBEPROXY_KUBECONFIG_FILE: $(yaml-quote "${WINDOWS_KUBEPROXY_KUBECONFIG_FILE}")
 WINDOWS_INFRA_CONTAINER: $(yaml-quote "${WINDOWS_INFRA_CONTAINER}")
+ENABLE_WINDOWS_PIGZ: $(yaml-quote "${ENABLE_WINDOWS_PIGZ}")
```

Comment:
I suggest WINDOWS_ENABLE_PIGZ to match the other Windows-specific variables here.